shared/lsif/ranges.ts

```diff
@@ -242,6 +242,33 @@ async function hasRangesQuery(queryGraphQL: QueryGraphQLFn<IntrospectionResponse
     return (await queryGraphQL(introspectionQuery)).__type?.fields.some(field => field.name === 'ranges') || false
 }
 
+const underlinesQuery = gql`
+    query Ranges($repository: String!, $commit: String!, $path: String!, $startLine: Int!, $endLine: Int!) {
```

Comment:
We may want to have a simpler graphql query that doesn't need to aggregate so much data if all we want is all ranges in a document.

Reply:
Yeah, I suspected there's a significantly faster way to get only the ranges. This was just a prototype I was hacking on to demonstrate a proof-of-concept to underline LSIF ranges. I'll make more noise about it another day once it's ready for review :)